end-to-end-computer-vision/configs/ingest_data.yaml

```diff
@@ -9,4 +8,5 @@ steps:
   upload_labels_to_label_studio:
     enable_cache: False
     parameters:
-      ls_project_id: 8
+      ls_project_id: 1
+      # storage_type: s3
```

Comment:
Why the commented out parameter here? Can we remove it?

Reply:
I thought that it would help others that are using aws to know that there is a storage type parama that they need to add

Reply:
Changed it to `# storage_type: s3 # For AWS use "s3" instead of "gcp" (the default value)`

Reply:
awesome this is good